terraform/modules/rds/variables.tf

```diff
@@ -45,7 +45,7 @@ variable "rds_parameter_group_name" {
 }
 
 variable "rds_parameter_group_family" {
-  default = "postgres9.6"
+  default = "default.postgres11"
```

Comment:
Is this even necessary?  This was just referring to the default parameter group, can we just remove this all together and let Terraform and AWS go with the default themselves?